portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -25,6 +25,21 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-66154."
+	@priority = "5"
+	test AssetNoPoPupWindow {
+		UserBar.gotoDropdownItem(dropdownItem = "Account Settings");
+
+		AssertElementPresent.assertElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE");
+
+		var currentURL = Navigator.getCurrentURL();
+		var p_p_authParam = StringUtil.extractLast("${currentURL}", "=");
+
+		Navigator.openSpecificURL(url = "http://localhost:8080/en/group/control_panel/manage;'*prompt()*'?p_p_id=com_liferay_my_account_web_portlet_MyAccountPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_auth=${p_p_authParam}");
```

Comment:
Please use `${portalURL}` instead of `localhost:8080`, we need to make sure test works with instance.